backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -25,26 +34,71 @@
 
 	private final SubscriptionService subscriptionService;
 
-	public UsersController(SubscriptionModelAssembler subscriptionModelAssembler, SubscriptionService subscriptionService) {
+	private final SubscriptionModelMapper mapper;
+
+	public UsersController(SubscriptionModelAssembler subscriptionModelAssembler, SubscriptionService subscriptionService, SubscriptionModelMapper mapper) {
 		Assert.notNull(subscriptionModelAssembler, "subscriptionModelAssembler is required; it must not be null");
 		Assert.notNull(subscriptionService, "subscriptionService is required; it must not be null");
 		this.subscriptionModelAssembler = subscriptionModelAssembler;
 		this.subscriptionService = subscriptionService;
+		this.mapper = mapper;
 	}
 
-	@GetMapping({ "/{id}/subscriptions" })
+	@GetMapping({ "/{userId}/subscriptions" })
 	@Operation(summary = "Get all subscriptions for a user.", operationId = "get-user-subscriptions")
 	public CollectionModel<SubscriptionModel> getSubscriptionsByUserId(
-			@NotBlank(message = "id must not be null or blank")
-			@Parameter(description = "The id of the user.", example = "000000000", required = true)
-			@PathVariable String id) {
-		final var subscriptions = subscriptionModelAssembler.toCollectionModel(subscriptionService.getSubscriptionsByUserId(id));
-		final var selfLink = linkTo(methodOn(getClass()).getSubscriptionsByUserId(id)).withSelfRel();
+			@NotBlank(message = "user id must not be null or blank")
+			@Parameter(description = "The user id of the user.", example = "000000000", required = true)
+			@PathVariable String userId) {
+		final var subscriptions = subscriptionModelAssembler.toCollectionModel(subscriptionService.getSubscriptionsByUserId(userId));
+		final var selfLink = linkTo(methodOn(getClass()).getSubscriptionsByUserId(userId)).withSelfRel();
 		return subscriptionModelAssembler.wrapCollection(subscriptions, SubscriptionModel.class).add(selfLink);
+	}	
+
+
+	@PostMapping({ "/{userId}/subscriptions/" })
+	@Operation(summary = "Create a subscription for a user.", operationId = "user-subscriptions-create")
+	@ApiResponse(responseCode = "202", description = "The request has been accepted for processing.")
+	public void create(
+			@RequestBody(required = true)
+			SubscriptionModel createSubscriptionRequest,
+
+			@RequestParam(defaultValue = "true", required = false)
+			@Parameter(description = "If the request should be handled asynchronously.", schema = @Schema(allowableValues = { "false", "true" }, defaultValue = "true"))
+			String async) {
+		if (!BooleanUtils.toBoolean(async)) {
+			log.warn("Call to unsupported operation: create(async=false)");
+			throw new UnsupportedOperationException("synchronous processing not yet implemented; please set async=true");
+		}
+
+		final var subscription = mapper.toDomain(createSubscriptionRequest);
+		log.debug("Creating subscription: {}", subscription);
+		subscriptionService.create(subscription);
 	}
 
+	@PostMapping({ "/{userId}/subscriptions/{id}" })
```

Comment:
Bit of a nitpick, but I think to keep things unambiguous, this should be `/{userId}/subscriptions/{subscriptionId}`

Reply:
updated